plugins/org.jkiss.dbeaver.ui.editors.connection/src/org/jkiss/dbeaver/ui/dialogs/driver/DriverSelectViewer.java

```diff
@@ -494,4 +496,14 @@ protected boolean isLeafMatch(Viewer viewer, Object element) {
 
     }
 
+    private static class DriverInstalledFilter extends ViewerFilter {
+        @Override
+        public boolean select(Viewer viewer, Object parentElement, Object element) {
+            if (element instanceof DBPDriver driver) {
```

Comment:
Is it expected that it doesn't check for `DBWorkbench.isDistributed()`?

Reply:
IT checks this when adding the filter.